src/DeepRL.jl

```diff
@@ -25,8 +25,13 @@ export
     sample_action,
     n_actions,
     obs_dimensions,
-    render
+    render,
+    generate_info
 
+# XXX
+function generate_info(p::Union{MDP,POMDP}, s::S, a::A, sp::S) where {S,A}
```

Comment:
In my experience, it's better to just leave out the type parameters in functions like these. Just use
```julia
function generate_info(p::Union{MDP,POMDP}, s, a, sp)
```